### PR TITLE
update reference of get-pip.py file from mongo-sidecar image

### PR DESCRIPTION
**Pull request by A-kanksh-a**
## Change Overview

This PR is to update url for `get-pip.py` file which is being referred in `ghcr.io/kanisterio/mongo-sidecar` image as [https://bootstrap.pypa.io/3.5/get-pip.py](https://bootstrap.pypa.io/3.5/get-pip.py) is no longer supported

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #XXX

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -56,3 +56,6 @@ jobs:
     - stage: test
       script:
       - make golint
+    - stage: test
```
Comment (pavannd1) [inline .travis.yml:59]: Looks like this change is unrelated

File: .goreleaser.yml [hunk 2]
```diff
@@ -127,8 +127,6 @@ dockers:
   dockerfile: 'docker/kafka-adobes3Connector/image/adobeSink.Dockerfile'
   extra_files:
   - 'docker/kafka-adobes3Connector/image/adobe-monitorsink.sh'
-archives:
```
Comment (pavannd1) [inline .goreleaser.yml:130]: Same as below - change is unrelated

File: .travis.yml (hunk 1 above)
Comment (A-kanksh-a) [reply .travis.yml:59]: yeah, I would remove this, added this stage just to check the CI running this script `make release-snapshot`

Comment (pavannd1) [reply .travis.yml:59]: Got it 👍🏼

File: .goreleaser.yml (hunk 2 above)
Comment (A-kanksh-a) [reply .goreleaser.yml]: I have removed these changes from PR.

File: .travis.yml (hunk 1 above)
Comment (A-kanksh-a) [reply .travis.yml]: have removed this change.

**Review by pavannd1 — APPROVED** [review]
